src/components/Articles-Single-Card.jsx

```diff
@@ -31,14 +56,20 @@ const ArticlesSingleCard = () => {
             <button>
                 <h3>In {singleArticle.topic}</h3>
             </button>
-            <p>
-            {singleArticle.body}
-            </p>
+            <p>{singleArticle.body}</p>
             <img src={singleArticle.article_img_url} />
             <h3>{singleArticle.created_at}</h3>
-            <button>{singleArticle.comment_count} Votes</button>
+            <button onClick={handleDownVote}>
+                <TiThumbsDown />
+            </button>
+            <button>{optimisticVotes} Votes</button>
```

Comment:
It might be better if the votes is not a button. It's slightly confusing 'cause it's clickable but does nothing